src/features/ShowMovie/ui/ProjectionsList.tsx

```diff
@@ -1,31 +1,63 @@
-import { FC } from 'react';
+import { FC, useState } from 'react';
 
 import { Session } from 'entities/session/model/types';
-import { transformSessionsToCinemasWithTimeSlots } from '../utils/transformSessionsToCinemasWithTimeslots';
+import { getPeriodOfDay } from 'shared/utils/getTimeOfDay';
 
+import { transformSessionsToCinemasWithTimeSlots } from '../utils/transformSessionsToCinemasWithTimeslots';
 import { Projection } from './Projection';
+import { ProjectionFilters } from './ProjectionsFilters';
 
 interface ProjectionListProps {
     sessions: Session[];
     activeDate: Date | null;
 }
 
 export const ProjectionsList: FC<ProjectionListProps> = ({ sessions, activeDate }) => {
+    const [filters, setFilters] = useState({
+        city: '',
+        cinema: '',
+        time: ''
+    });
+
     const filteredSessions = sessions.filter(
         (s) => new Date(s.date).getTime() === activeDate?.getTime()
     );
+
     const timeSlotsByCinema = transformSessionsToCinemasWithTimeSlots(filteredSessions);
 
     return (
         <div className="mx-20 my-12 font-effra">
-            {Object.entries(timeSlotsByCinema).map((timeSlotByCinema) => (
-                <Projection
-                    key={timeSlotByCinema[0]}
-                    city={timeSlotByCinema[1].city}
-                    cinema={timeSlotByCinema[0]}
-                    timeSlots={timeSlotByCinema[1].timeSlots.sort()}
-                />
-            ))}
+            <ProjectionFilters items={Object.entries(timeSlotsByCinema)} setFilters={setFilters} />
```

Comment:
you call Object.entries(timeSlotsByCinema) 2 times here and on the next line, in such cases create separate const and use it